src/xdr/Stellar-transaction.x

```diff
@@ -221,6 +222,23 @@ struct ManageDataOp
     DataValue* dataValue;   // set to null to clear
 };
 
+/* Bump Sequence
+
+    increases the sequence to a given level
+
+    Result: BumpSequenceResult
+*/
+
+struct BumpSeqValidRange {
+    SequenceNumber min;
+    SequenceNumber max;
+};
+struct BumpSequenceOp
+{
+    SequenceNumber bumpTo;
+    BumpSeqValidRange* range;
+};
```

Comment:
Wondering if there are use cases in which only min or only max is needed. And then maybe we should interpret max = 0 as no max. Like in TimeBounds:
```xdr
struct TimeBounds
{
    uint64 minTime;
    uint64 maxTime; // 0 here means no maxTime
};
```

Reply:
This should be equivalent:

`max = std::numeric_limits<uint64_t>::max()`

Reply:
Yeah, I know but we have special case in `TimeBounds` and need to use uint64 max here. So wondering if it shouldn't be consistent.

Reply:
I think we should go with 0 for no max value for consistency

Reply:
the use of 0 instead of `std::numeric_limits<uint64_t>::max()` is redundant and confusing (and makes everybody write more code).
I would also ditch the optional attribute on the range and always specify it - the only reason we made it optional for TimeBounds is because we can save a few bytes with every transaction (very few people use time bounds).